Portal_Perfomance_Employes/Controllers/EmployeController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Portal_Perfomance_Employes.Data;
+using Portal_Perfomance_Employes.Models;
+
+namespace Portal_Perfomance_Employes.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EmployeController : ControllerBase
+    {
+        private EmployesDB _employesDB = new EmployesDB();
+
+        public EmployeController()
+        {
+            _employesDB.PopularEmployesDB();
+        }
+
+
+        [HttpGet]
+        public async Task<IEnumerable<Employe>> Get() => _employesDB.employes.ToList();
+        
+        [HttpGet("{id}")]
+        public IActionResult GetbyId(int id)
```

Comment:
Should we make this async like the other calls?

Reply:
About the async it is not allowed to use async on method IActionResult. But if it makes sense to change the type of the method so yes we should use async.

Reply:
Yes, but it's fine to use async on a `Task<IActionResult>` :)

Reply:
Ah nice, i changed the method to be async too.